core/runtime.rs

```diff
@@ -81,6 +83,7 @@ pub struct JsRuntime {
   snapshot_creator: Option<v8::SnapshotCreator>,
   has_snapshotted: bool,
   allocations: IsolateAllocations,
+  modules: Vec<Box<dyn JsRuntimeModule>>,
```

Comment:
I think we should use a different word than "module" because it might be confused with ES modules.

Currently we use the term "op_crates" but maybe we don't want to explicitly tie this to the rust crate concept. Maybe something like "op_package" ?

Reply:
Maybe a RuntimeExtension?

Reply:
Both are good candidates, I prefer `RuntimeExtension` but `OpBundle` isn't bad either. `OpBundle` is a little more specific and familiar given "op_crates", but `RuntimeExtension` is a better fit since runtime extensions can be pure JS too.

I could easily imagine the following in the docs:
> Runtime extensions implement the `deno_core::JsRuntimeExtension` interface(/trait), they extend the runtime's features by adding JS to the global scope and providing ops. Extensions are typically pure JS, a mix of JS & matching ops or stateful-ops & op-middleware in more advanced cases.

I'll continue with the current naming and reword on the "polish pass" once we've reached consensus.

Reply:
RuntimeExtension sounds good to me

Reply:
I'm strongly in favor of `op_crates`, this term is already prevalent

Reply:
But not all op_crates contain ops 😉 (op_crates/web)

Reply:
Sure... I just want to avoid situation where the same thing has two distinct names

Reply:
`op_crates` isn't a bad name but it's a little too specific IMO:
- Extensions can be pure JS
- Extensions can be grouped (`runtime` will have many small op-extensions grouped with MultiExtensions)
- Plugins aren't op_crates but should return extensions (in the future)

There's no "perfect" candidate suggested so far, all have tradeoffs on familiarity/ambiguity/descriptiveness.

Obviously we could rename `op_crates` to `ext_crates` but that could be unnecessarily disruptive...

Reply:
Why not `Extension` (drop the `Runtime`)? and potentially op_crates->extensions later.